README.md

```diff
@@ -5,20 +5,30 @@
+Karpenter is an open-source node provisioning project built for Kubernetes.
+Its goal is to improve the efficiency and cost of running workloads on Kubernetes clusters.
+Karpenter works by:
 
-Karpenter is a node lifecycle management solution. It observes incoming pods and launches the right instances for the situation. Instance selection decisions are intent based and driven by the specification of incoming pods, including resource requests and scheduling constraints.
+* **Watching** for pods that the Kubernetes scheduler has marked as unschedulable
+* **Evaluating** scheduling constraints (resource requests, nodeselectors, affinities, tolerations, and topology spread constraints) requested by the pods
+* **Provisioning** nodes that meet the requirements of the pods
+* **Scheduling** the pods to run on the new nodes
+* **Removing** the nodes when the nodes are no longer needed
 
-It's responsible for:
-- **Launching** nodes for unschedulable pods
-- **Replacing** existing nodes to improve resource utilization
-- **Terminating** nodes if outdated or no longer needed
-- **Draining** nodes gracefully before preemption
+For most use cases, a cluster’s capacity can be managed by a single Karpenter Provisioner.
+However, you can define multiple Provisioners, enabling use cases like isolation, entitlements, and sharding.
+Using a combination of defaults and overrides, Karpenter determines the availability zone, instance type, capacity type, machine image, and scheduling constraints for pods it manages.
 
-For most use cases, the entirety of a cluster’s capacity can be managed by a single Karpenter [Provisioner](README.md). Availability zone, instance type, capacity type, machine image, and scheduling constraints are automatically determined by the controller using a combination of defaults and overrides. Additionally, you can define multiple Provisioners, enabling use cases like isolation, entitlements, and sharding.
+Karpenter optimizes for scheduling latency and utilization efficiency using two complementary control loops:
 
-Karpenter optimizes for scheduling latency and utilization efficiency using two complementary control loops. First, is the allocator, a fast-acting latency-sensitive controller responsible for ensuring that incoming pods are scheduled as quickly as possible. Second, is the reallocator, a slow-acting cost-sensitive controller that replaces nodes as pods requests and capacity prices shift over time. Together, they maximize the availability and efficiency of your cluster.
+* The *allocator* is a fast-acting latency-sensitive controller that is responsible for ensuring that incoming pods are scheduled as quickly as possible.
+* The *reallocator* is a slow-acting cost-sensitive controller that replaces nodes as pod requests and capacity prices shift over time.
```

Comment:
I didn't know about this breakdown, great info.

Reply:
I didn't either. It was in the original README, so I left it.